src/components/ArticlesList.jsx

```diff
@@ -2,32 +2,36 @@ import Article from "./Article"
 import { getArticles } from "../utils/api"
 import {useSearchParams} from "react-router-dom"
 import { useEffect, useState } from "react"
+import Error from "./Error"
```

Comment:
One bit of error handling that's missing is when the user tries to submit a comment while offline, this currently causes an error and crashes the site
Could you please add functionality to catch this error and provide relevant feedback to the user?